tests/components/manual/test_alarm_control_panel.py

```diff
@@ -1396,7 +1397,7 @@ async def test_armed_custom_bypass_with_specific_pending(hass):
 
     await common.async_alarm_arm_custom_bypass(hass)
 
-    assert STATE_ALARM_PENDING == hass.states.get(entity_id).state
+    assert STATE_ALARM_ARMING == hass.states.get(entity_id).state
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5411564)